library/network_connections.py

```diff
@@ -1148,6 +1148,61 @@ def connection_create(self, connections, idx, connection_current=None):
                     s_ip4.add_route(rr)
                 else:
                     s_ip6.add_route(rr)
+            for r in ip["routing_rules"]:
+                routing_rule = NM.IPRoutingRule.new(r["family"])
+                if r["priority"] is not None:
+                    NM.IPRoutingRule.set_priority(routing_rule, r["priority"])
+                if r["action"]:
+                    for i, action in enumerate(
+                        [
+                            "to-table",
+                            "blackhole",
+                            "prohibit",
+                            "unreachable",
+                        ]
+                    ):
+                        if action == r["action"]:
+                            # check the link below for the enum value of supported action
+                            # https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/uapi/linux/fib_rules.h#n88
```

Comment:
use the URL that contains the commit ID to which you point at. Otherwise, `#n88` (or the entire file) is not guaranteed to be there in the future.